### ISSUE TITLE
Dialog component DOM differences between server and client on Nuxt

### ISSUE THREAD

**Issue by bpeab** [liip/chusho#31]
The Dialog component cause the following warning when used on Nuxt :

```
[Vue warn]: The client-side rendered virtual DOM tree is not matching server-rendered content. This is likely caused by incorrect HTML markup, for example nesting block-level elements inside <p>, or missing <tbody>. Bailing hydration and performing full client-side render.
```

This warning is likely because of some rendering happening on the server-side which shouldn't happen when Chusho is loaded in `client` mode. Using the `client-only` Nuxt component doesn't help neither.

Also, I tried to add the `portal-target` myself with and without `portal-vue` and its module installed but it didn't help. 

Finally, I tried to remove the `portal` detection that Chusho uses to add the `portal-target` if it doesn't exists but it also didn't help.

**Comment by LeBenLeBen** [comment]
That’s weird, I did some testing with Vue CLI in SSR and the documentation is also server-side rendered and didn't got any warning like that. That might be specific to Nuxt… I'll investigate asap!

**Comment by LeBenLeBen** [comment]
Released in 0.1.0-alpha.3
